shoop/core/models/_contacts.py

```diff
@@ -92,9 +96,25 @@ def __init__(self, *args, **kwargs):
             kwargs.setdefault("tax_group", self.default_tax_group_getter())
         super(Contact, self).__init__(*args, **kwargs)
 
+    def save(self, *args, **kwargs):
+        super(Contact, self).save(*args, **kwargs)
+        if self.default_contact_group_identifier:
+            self.groups.add(self.get_default_contact_group())
+
+    def get_default_contact_group(self):
+        obj, created = ContactGroup.objects.get_or_create(
+            identifier=self.default_contact_group_identifier,
+            defaults={
+                "name": self.default_contact_group_name
```

Comment:
This will create the group with the name in some arbitrary language, depending on which i18n context is active during the request that creates the first contact of each type. Not sure that's what's wanted.